tools/scaling/node_to_node/unstructured_mesh/strong/64rpn/strong_scaling.geo

```diff
@@ -0,0 +1,63 @@
+// Parameters
+cube_size = 10.0;          // Side length of the cube
+
+// Calculate characteristic length based on the desired number of tetrahedra
+approx_element_size = cube_size / 39;
```

Comment:
`gmsh` takes a command line argument to prescribe a variable. So, you can use just one .geo file and parameterize it from a command line.

If you changed this line to:
```
approx_element_size = cube_size /  divisor;
```
then, you could generate the mesh like this:
```
gmsh -3 -setnumber divisor 39 strong_scaling.geo -o mesh.msh
```

Reply:
Oh... now that's nice. I'll take one `.geo` file and write a script that generates all of the runs.

Reply:
Fixed!